.travis.yml

```diff
@@ -10,13 +10,13 @@ rvm:
   - jruby-9.1.5.0 # is precompiled per http://rubies.travis-ci.org/
 
 jdk:
-  - oraclejdk8
+  - openjdk8
 
 before_install:
-  - gem update --system
-  - rvm @global do gem uninstall bundler -a -x
+  - gem update --system 2.6.8
+  - if [ "$RUBY_VERSION" != "jruby-9.1.5.0" ] ; then update_rubygems ; fi 
```

Comment:
Ruby version will never be jruby

Reply:
Actually, on Travis, it is. See this build where I called `env` - line 395 shows what `RUBY_VERSION` is.
https://travis-ci.org/rails-api/active_model_serializers/jobs/652094232#L395

There is no `RUBY_ENGINE` defined in travis.

Reply:
That said, if you have another idea how to do this, I'm completely open to it

Reply:
sorry for constant pinging, but looks like this is fine as it is? @bf4

Reply:
ah, my bad. that's shell env, not ruby